javaslang/src/main/java/javaslang/collection/HashSet.java

```diff
@@ -559,7 +559,7 @@ public boolean contains(T element) {
             return empty();
         } else {
             final HashArrayMappedTrie<U, U> that = foldLeft(HashArrayMappedTrie.empty(),
-                    (tree, t) -> addAll(tree, mapper.apply(t)));
+                                                            (tree, t) -> addAll(tree, mapper.apply(t)));
```

Comment:
It is ok for me as-is. I recently did a few changes to the formatter regarding indentation. I will apply the new rules to the whole code-base soon, so this space actually does not matter.
Thank you!